tensorboard/backend/application.py

```diff
@@ -345,9 +365,14 @@ class are WSGI applications.
     clean_path = _clean_path(parsed_url.path, self._path_prefix)
 
     # pylint: disable=too-many-function-args
-    if clean_path in self.data_applications:
-      return self.data_applications[clean_path](environ, start_response)
+    if clean_path in self.exact_routes:
+      return self.exact_routes[clean_path](environ, start_response)
     else:
+      for path_prefix in self.prefix_routes:
+        if clean_path.startswith(path_prefix):
+          return self.prefix_routes[path_prefix](environ, start_response)
+        print(clean_path, ' does not start with ', path_prefix)
```

Comment:
If we keep this around, it should be a logging statement, using a format
string so that it does not format the arguments when the message
wouldn’t be printed anyway.

Reply:
Sorry, I just overlooked this debugging cruft.